src/lib/mina.ts

```diff
@@ -318,8 +324,37 @@ function LocalBlockchain({
             slot: networkState.globalSlotSinceHardFork.toString(),
           });
         }
-      });
 
+        // actions/sequencing events
+        let latestActionsHash =
+          ledger.getAccount(
+            PublicKey.fromBase58(addr),
+            Ledger.fieldOfBase58(tokenId)
+          )?.zkapp?.sequenceState[0] ?? Events.emptySequenceState();
```

Comment:
I don't think getting the latestActionsHash from the ledger here is correct in the most general case, where there are multiple parties from the same account that have sequence events, in one transaction

Reply:
good call, this also fixed the `+2` thing!